### PR TITLE
Update Apperian endpoints to replace deprecated ones

### PR DESCRIPTION
**Pull request by carlos-enciso**

### CODE REVIEW

File: lib/upload-apperian.js [hunk 1]
```diff
@@ -99,39 +89,34 @@ var main = (function () {
...
+	var deviceId = argv.deviceid || DEVICE_ID;
 
 	/**
-	 * @method executeMethod
+	 * @method executeRequest
 	 * @private
-	 * Executes method for Apperian
-	 * @param {String} _method Method to execute
+	 * Executes request for Apperian
+	 * @param {String} _enpoint Request to execute
```
Comment (pablog178) [inline lib/upload-apperian.js]: endpoint

File: lib/upload-apperian.js [hunk 2]
```diff
@@ -174,11 +160,50 @@ var main = (function () {
...
+			headers: {
+				'X-TOKEN': transactionToken
+			},
+			url: WEBSERVICE_URL + METHOD_UPDATE,
+			formData: formData
+		}, function (_error, _httpResponse, _body) {
+			doLog && spinner.stop();
+			if (_error) {
+				console.error('\nUpload failed:', _error);
+				throw new Error('Upload failed:', _error);
```
Comment (pablog178) [inline lib/upload-apperian.js:201]: Print the `error.message` property so it's easier to read

**Review by pablog178 — CHANGES_REQUESTED** [review]
Very nice work, thanks @carlos-enciso !!

**Review by pablog178 — APPROVED** [review]

**Review by pablog178 — APPROVED** [review]

File: lib/upload-apperian.js [hunk 3]
```diff
@@ -190,28 +215,31 @@ var main = (function () {
...
-				break;
-			case 'android':
-				type = 'APK';
-				break;
-			case 'microsoft':
-				type = 'APPX';
-				break;
-			default:
+				case 'ios':
+					type.push(1); // iOS
```
Comment (ccavazos) [inline lib/upload-apperian.js:226]: I'd probably have these as constants and add a code comment where they are coming from

**Review by ccavazos — APPROVED** [review]

**Pull request merged**